roles/ci_nmstate/molecule/default/converge.yml

```diff
@@ -41,6 +41,7 @@
           - "!all"
           - "!min"
           - network
+          - dns
```

Comment:
isn't "dns" a subset of "network" ?

Reply:
Cannot make it to work without this addition. Agree that, in theory, it's part of the network facts.